config/initializers/sidekiq.rb

```diff
@@ -6,4 +6,16 @@
   Sidekiq.configure_client do |config|
     config.redis = {ssl_params: {verify_mode: OpenSSL::SSL::VERIFY_NONE}}
   end
+elsif ENV["KAMAL_SETUP"] == "true"
```

Comment:
What would be a good place for a global `kamal?` helper?

Reply:
Probably here: https://github.com/bullet-train-co/bullet_train-core/blob/520133dab93d1b549c22f128bba190e5b9bf8516/bullet_train/lib/bullet_train.rb